package.json

```diff
@@ -4,10 +4,10 @@
   "description": "Simple HTML5 charts using the canvas element.",
   "version": "3.0.0-alpha",
   "license": "MIT",
-  "jsdelivr": "dist/Chart.min.js",
-  "unpkg": "dist/Chart.min.js",
-  "main": "dist/Chart.js",
-  "module": "dist/Chart.esm.js",
+  "jsdelivr": "chart.min.js",
```

Comment:
why do we need to move the bundles? they can remain in the dist folder, since the `package.json` is referencing them

Reply:
that's true, those don't need to be moved